decktoimage/decktoimage.py

```diff
@@ -25,7 +25,7 @@
 star = 'resources/star.png'
 
 deck_font = 'resources/Belwe-Bold.ttf'
-name_font = 'resources/NotoSansCJK-Bold.ttc'
+name_font = 'resources/Belwe-Bold.ttf'
```

Comment:
I think this font doesn't support Asian characters. This might matter in some use cases, but I don't think enough enough people use this library for it to matter.

Reply:
I think as it can only export English deck images, it makes sense if it also only supports English deck names. But anyway I'm ok if you think the font is unsuitable.